src/app/App.test.tsx

```diff
@@ -174,7 +174,7 @@ describe("App", () => {
 	});
 
 	it("should render application error if the app fails to boot", async () => {
-		const envSpy = jest.spyOn(env, "boot").mockImplementation(() => {
+		const envSpy = jest.spyOn(Environment.prototype, "boot").mockImplementation(() => {
```

Comment:
🤨

Reply:
i don't have access to the specific env instance that needs to be mocked for this test, so changing all of them is the only option - unless there is a better approach i'm unaware of.